data/teams.py

```diff
@@ -1,112 +1,47 @@
-# the following were created like so
-# import statsapi
-# _teams = statsapi.get('teams', {'sportIds':1})['teams']
-# TEAM_ID_ABBR = {t["id"]: t["abbreviation"] for t in _teams}
-# TEAM_ID_NAME = {t["id"]: t["teamName"] for t in _teams}
-# TEAM_NAME_ID = {t["teamName"]: t["id"] for t in _teams}
-
-# Can be customized, but will require changing in colors/teams.json as well
```

Comment:
Should we still mention customization? I know at least one user who uses names like “Cubbies” and “Yanks” instead